src/test/java/org/sqlite/ResultSetTest.java

```diff
@@ -14,11 +14,14 @@
 import java.sql.Statement;
 import java.sql.Time;
 import java.sql.Timestamp;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
-public class ResultSetTest {
+class ResultSetTest {
 
```

Comment:
why all the changes of visibility ?

Reply:
Since Junit 5, it supports classes which has package private access, so it's redundant, and I have Sonarlint installed, it complaints the modifier...